src/runtime/less/ts-buttons.less

```diff
@@ -190,6 +190,7 @@
 	&.ts-tertiary {
 		line-height: @ts-unit - 2;
 	}
+	display: inline-block;
```

Comment:
I think that this should be moved to the `.mixin-button()` in the top of the file, though, so that *anything* can be made to behave like a button (like in this case the `a` that wasn't working before).